python/tests/conftest.py

```diff
@@ -97,9 +92,25 @@ def create_test_users(
         sdk.delete_user(user_id)
 
 
+@pytest.fixture(scope="session", params=["3.1", "4.0"])
+def api_version(request):
+    return request.param
+
+
 @pytest.fixture(scope="session")
-def sdk():
-    # TODO multiplex this fixture for both init31 and init40
-    sdk = looker_sdk.init31("../looker.ini")
+def sdk(api_version: str):
+    if api_version == "3.1":
+        sdk = looker_sdk.init31("../looker.ini")
+    elif api_version == "4.0":
+        sdk = looker_sdk.init40("../looker.ini")  # type: ignore
```

Comment:
declare `sdk: Union[methods31:Looker31SDK, methods40.Looker40SDK]` above and you should be able to remove the `# type: ignore`